dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/postgres/V6.0_2015.03.07__hibernate_migration.sql

```diff
@@ -193,7 +193,7 @@ ALTER TABLE bundle2bitstream ALTER COLUMN bitstream_id SET NOT NULL;
 ALTER TABLE bundle2bitstream DROP COLUMN bundle_legacy_id;
 ALTER TABLE bundle2bitstream DROP COLUMN bitstream_legacy_id;
 ALTER TABLE bundle2bitstream DROP COLUMN id;
-ALTER TABLE bundle2bitstream add primary key (bitstream_id,bundle_id);
+ALTER TABLE bundle2bitstream add primary key (bitstream_id,bundle_id,bitstream_order);
```

Comment:
Right, I understand. I was more noting there's two ways to achieve this database migration:
1) Edit the existing migration script (as you did). This means anyone who had previously run the old migration script will need to drop the old database and start fresh (as Flyway won't rerun this migration and correct their DB schema).
2) OR, create a new migration script specifically for this new schema change. This would mean developers could just run the new migration on their existing pre-6.0 database.

In this scenario, going with option #1 is OK (as it only affects a few of us and this work isn't over on "master" yet).  We just need to tell everyone to drop their existing pre-6.0 databases, and start fresh.  

So, my comment was just a reminder that we need to let everyone know to drop their existing pre-6.0 databases once this PR is merged :)